http/src/main/resources/reference.conf

```diff
@@ -383,3 +392,8 @@ ui {
   terraLabel = "saturnAutoCreated"
   allOfUsLabel = "all-of-us"
 }
+
+async-task-processor {
```

Comment:
Just a nit: I notice that all our other configs are camelCase :)

Reply:
yea...I've been doing camelCase in leonardo as well...I just thought if we're serious about casing, I think it's recommended to do hyphen-separated style, see https://github.com/lightbend/config/blob/master/HOCON.md#hyphen-separated-vs-camelcase...also this is default style pureconfig uses, which I'm thinking maybe one day we'll move over to pureconfig :P